src/components/order-service/order-service.controller.ts

```diff
@@ -312,6 +313,34 @@ export class OrderServiceController implements angular.IController {
     this.ctrl.parameterSchema = schema;
   }
 
+  private updateParameterForm() {
+    let formDefinition: any = _.get(this, 'ctrl.selectedPlan.externalMetadata.schemas.service_instance.create.form_definition');
+    this.removeUnsupportedParameterFormItems(formDefinition);
+    this.ctrl.parameterForm = formDefinition || ['*'];
+  }
+
+  private removeUnsupportedParameterFormItems(formItems: any) {
```

Comment:
It would be better to move this in the catalog-parameters component so that we will be able to reuse it for bind parameters.